@here/harp-vectortile-datasource/lib/VectorTileDataEmitter.ts

```diff
@@ -1328,7 +1328,7 @@ export class VectorTileDataEmitter {
             : isFilled
             ? fillTechnique.lineWidth ?? 0.0
             : 0.0;
-        const hasEdges = edgeWidth > 0.0;
+        const hasEdges = typeof edgeWidth === "number" ? edgeWidth > 0.0 : true;
```

Comment:
What else could edgeWidth be if not a number? An expression? Couldn't we check for an expression if it can ever evaluate to `> 0.0` for the given tile level?

Reply:
It could be an Expression... 
yes, we could probably do so, but not sure its worth it, as the whole code seems to be very unstable, the edgeWidth is never evaluated after that, and if there is no edges, in Safari, nothing will be rendered at all.

Reply:
I had a look, seems its not much hazzle to do so, I ll push an update